pkg/web/templates/partials/transaction/transaction_info.html

```diff
@@ -15,7 +15,7 @@ <h1 class="text-xl mb-1 font-bold text-gray-900 lg:text-2xl">
           {{ if $canMngTrans }}
            {{ if and (eq .Source "remote") (ne .Status "completed") }}
             <a href="/transactions/{{ .ID }}/accept" class="p-1 rounded w-32 bg-success font-semibold text-white text-center md:p-2 hover:bg-success/80">Accept</a>
-            <button onclick="transaction_rejection_modal.showModal()" class="p-1 rounded w-32 bg-warning font-semibold text-white md:p-2 hover:bg-warning/80">Reject</button>
+            <a href="/transactions/{{ .ID }}/info" class="p-1 rounded w-32 bg-warning font-semibold text-white text-center md:p-2 hover:bg-warning/80">Reject</a>
```

Comment:
When attempting to reject a transaction on the `/transactions` page, there was an error because the `ID` wasn't included in the request. I'll create a separate story to resolve that, so decided to navigate users to the info page for now.